soc/arm/nxp_imx/rt/dts_fixup.h

```diff
@@ -25,13 +25,6 @@
 #define DT_IRQ_ETH1_COMMON			DT_NXP_KINETIS_ETHERNET_402D4000_IRQ_0
 #define DT_IRQ_ETH1_IEEE1588_TMR		DT_NXP_KINETIS_PTP_402D4000_PTP_IRQ_0
 
-#define DT_USBD_MCUX_EHCI_NAME        DT_NXP_KINETIS_USBD_402E0000_LABEL
-#define DT_USBD_MCUX_EHCI_IRQ         DT_NXP_KINETIS_USBD_402E0000_IRQ_USB_OTG
-#define DT_USBD_MCUX_EHCI_IRQ_PRI     DT_NXP_KINETIS_USBD_402E0000_IRQ_0_PRIORITY
-#define DT_USBD_MCUX_EHCI_BASE_ADDRESS    DT_NXP_KINETIS_USBD_402E0000_BASE_ADDRESS
-#define DT_USBD_MCUX_EHCI_NUM_BIDIR_EP    DT_NXP_KINETIS_USBD_402E0000_NUM_BIDIR_ENDPOINTS
```

Comment:
DT_USBD_MCUX_EHCI_NUM_BIDIR_EP  is used in ../modules/hal/nxp/mcux/middleware/usb/device/usb_device_ehci.h

Reply:
Thanks for pointing that out:
https://github.com/zephyrproject-rtos/hal_nxp/pull/37
https://github.com/zephyrproject-rtos/zephyr/pull/24412